src/main/java/Calculator.java

```diff
@@ -0,0 +1,75 @@
+public class Calculator {
+
+    public static final String plus = "+";
+    public static final String minus = "-";
+    public static final String multiple = "*";
+    public static final String divide = "/";
+
+    public int add(int num1, int num2) {
+        return num1 + num2;
+    }
+
+    public int minus(int num1, int num2) {
+        return num1 - num2;
+    }
+
+    public int multiply(int num1, int num2) {
+        return num1 * num2;
+    }
+
+    public int divide(int num1, int num2) {
+        return num1 / num2;
+    }
+
+    public String execute(String mathExpression) throws IllegalArgumentException{
+
+        String[] splitMath = checkMathExpression(mathExpression);
+
+        for(int i = 0; i < splitMath.length - 1; i = i+2) {
+            int num1 = Integer.parseInt(splitMath[i]);
+            int num2 = Integer.parseInt(splitMath[i+2]);
+            String operator = splitMath[i+1];
+            checkOperator(operator);
+            splitMath[i+2] = executeCalculator(operator, num1, num2);
+        }
+
+        return splitMath[splitMath.length - 1];
+
+    }
+
+    public String[] checkMathExpression(String mathExpression) {
+
+        if(mathExpression == null || mathExpression.equals("")) {
```

Comment:
```suggestion
        if (mathExpression == null || mathExpression.equals("")) {
```

if나 for와 같은 연산자 후에 공백을 사용하지 않는 것이 훈섭님의 컨벤션일까요!?
해당 미션을 진행하는 동안은 자바 코드 컨벤션을 지키며 작성하는게 좋을 것 같은데 훈섭님의 생각은 어떠신가요 😄 

컨벤션은 https://google.github.io/styleguide/javaguide.html 또는 https://myeonguni.tistory.com/1596를 참고해주세요 :)